static/app/components/replays/virtualizedGrid/useDetailsSplit.tsx

```diff
@@ -52,12 +52,15 @@ export default function useDetailsSplit({
   // `initialSize` cannot depend on containerRef because the ref starts as
   // `undefined` which then gets set into the hook and doesn't update.
   const initialSize = Math.max(150, window.innerHeight * 0.4);
+  const [containerSize, setContainerSize] = useState(initialSize);
 
-  const {size: containerSize, ...resizableDrawerProps} = useResizableDrawer({
+  const resizableDrawerProps = useResizableDrawer({
     direction: 'up',
     initialSize,
     min: 0,
-    onResize: () => {},
+    onResize: (newSize: number) => {
```

Comment:
types can be inferred:

```suggestion
    onResize: (newSize) => {
```